auth-web/src/components/auth/manage-business/AffiliationAction.vue

```diff
@@ -47,19 +47,15 @@
             </v-btn>
           </template>
           <v-list>
-            <template v-if="!!item.affiliationInvites && isCurrentOrganization(item.affiliationInvites[0].fromOrg.id)">
-              <v-list-item
-                v-if="item.affiliationInvites[0].status !== AffiliationInvitationStatus.Accepted &&
-                  item.affiliationInvites[0].type === AffiliationInvitationType.REQUEST"
-                class="actions-dropdown_item my-1"
-                @click="openNewAffiliationInvite(item)"
-              >
-                <v-list-item-subtitle>
-                  <v-icon small>mdi-refresh</v-icon>
-                  <span class="pl-1">New Request</span>
-                </v-list-item-subtitle>
-              </v-list-item>
-            </template>
+            <v-list-item
+              v-if="showAffiliationInvitationNewRequestButton(item)"
+              class="actions-dropdown_item my-1"
+              @click="openNewAffiliationInvite(item)"
+            >
+              <v-list-item-subtitle>
+                <span class="pl-1">New Request</span>
```

Comment:
No icon?

Reply:
No, UI design has no icons for actions

Reply:
Interesting, Ethan said otherwise

Reply:
<img width="728" alt="image" src="https://github.com/bcgov/sbc-auth/assets/7550170/81c9ead2-bd8a-4144-b958-2e74dab25940">

Reply:
Fairly certain this isn't correct 

@ethantspitt could you comment?

Reply:
He tried to comment - but github is having issues, basically said you're working off an older design. I'll post the screenshot after RocketChat starts working again

Reply:
![image](https://github.com/bcgov/sbc-auth/assets/3484109/18d1c818-4308-4a12-94ba-839982fb3b27)

Reply:
You got the icon off of Scott, so we're good

Reply:
re-added icon